### ISSUE TITLE
Restyling: Renewal of the Cart Landing Page

### ISSUE THREAD

**Issue by msaroshali** [Xceptance/posters-demo-store#26]

**Comment by msaroshali** [comment]
Redesigning the page has led to some change in the ajax seemingly, 

1. The page now to seems reload after deleting a product.
2. Updating the quantity updates prices in the cart summary right away, but the updated price in the item column is visible only once the page is forced to reload.

Creating a separate issue for these problem.
